Global/JetBrains.gitignore

```diff
@@ -29,6 +29,7 @@
 
 # IntelliJ
 /out/
+.idea/
```

Comment:
The link mentioned outlines the things that should be shared, and they should be already defined.

I'm 👎 on ignoring everything under `.idea`.